packages/geo/src/lib/print/shared/print.service.ts

```diff
@@ -472,6 +494,7 @@ export class PrintService {
             'print'
           );
         }
+        this.addCanvas(doc, canvasOverlayHTML, margins); // this adds scales and attributions
```

Comment:
J'ai une erreur :
ERROR TypeError: Cannot read property 'toDataURL' of undefined
    at PrintService.addCanvas